fr-ln/jw300-baseline/baseline.ipynb

```diff
@@ -0,0 +1,6194 @@
+{
```

Comment:
<p>what do you mean by "french swc" test set? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/masakhane-io/masakhane/pull/73/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>It's should be french Lingala, I copy and paste from another notebook..</p><p>I will definitely update...</p>

Reply:
Fixed...🤦🏾‍♂️